modules/op-monitoring-pipeline-run-status-using-pipelines-as-code.adoc

```diff
@@ -25,11 +25,11 @@ When {pac} detects an error in one of the tasks of a pipeline, a small snippet c
 [discrete]
 .Annotations for log error snippets
 
-In the {pac} config map, if you set the `error-detection-from-container-logs` parameter to `true`, {pac} detects the errors from the container logs and adds them as annotations on the pull request where the error occurred.
+In the `TektonConfig` custom resource, under `pipelinesAsCode` and `settings`, if you set the `error-detection-from-container-logs` parameter to `true`, {pac} detects the errors from the container logs and adds them as annotations on the pull request where the error occurred.
```

Comment:
```suggestion
If you set the `error-detection-from-container-logs` parameter to `true`, {pac} detects the errors from the container logs and adds them as annotations on the pull request where the error occurred. You can set this parameter in the `TektonConfig` custom resource (CR), in the `pipelinesAsCode` and `settings` sections.
```
splitting into two sentences for readability suggested

Reply:
I cannot take this suggestion as a hanging "parameter" does not seem acceptable to me, but this needs changing for the "spec" anyway.